tensorflow/c/c_api.h

```diff
@@ -1128,6 +1128,15 @@ TF_CAPI_EXPORT extern void TF_FinishWhile(const TF_WhileParams* params,
 // called after a successful TF_NewWhile() call.
 TF_CAPI_EXPORT extern void TF_AbortWhile(const TF_WhileParams* params);
 
+// Adds operations to compute the partial derivatives of sum of `y`s w.r.t `x`s.
+//
+// This method is the equivalent of calling TF_AddGradientsWithPrefix with a
+// nullptr prefix (which will create all gradients operations under "gradients/"
```

Comment:
Ah, this isn't technically correct, since if there is naming clash it will create it under `gradient_1/` or `gradients_2/` etc. no?

I'd be okay just removing this comment, or maybe saying
```
See TF_AddGradientsWithPrefix, which provides a means to control the names of operations added to the graph in order to compute gradients.
```

or something like that?